### ISSUE TITLE
Pixel Polishes on the sim-a11y-view template

### ISSUE THREAD

**Issue by zepumph** [phetsims/chipper#610]
From https://github.com/phetsims/a11y-research/issues/56

There are a few more things that still need work on this html page, and here is a great place for them to get done.

Here is one of @terracoda's last comments from the previous issue:

>@zepumph and @jessegreenberg, Looks great! Not sure if the emphasis element is used in different contexts in different sims. If it is an element that we can use consistantly for marking parameters, it would be really great. 

>In this sim, the highlighted green emphasis elements give a nice view of what content actually changes, making it very clear that this sim has a large amount of static content, with several embedded dynamic parameters.

>**Regarding actual style**
We need either a bit more line-height in general, and/or a little bottom padding on the list items. Either or both these style rules will prevent the overlapping of the parameters which makes the double-line bottom border disappear on some parameters.

>**Question:** Is there a global font-size/ light-height definition for this template? And, if not, we could try that.

>CSS3 has a new` rem` unit which means "root em." We might want to try that.

>- [2015 CSS Tricks articlel/](https://css-tricks.com/rem-global-em-local/)
>- [2011 Snook article](https://snook.ca/archives/html_and_css/font-size-with-rem)

@jessegreenberg and I think that if any major work is to be done to this html page, then we should talk with @emily-phet to make sure our priorities are aligned.

**Comment by terracoda** [comment]
For my reference, the latest visual PDOM, or A11y View of Ohm's Law:
[http://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-dev.12/ohms-law-a11y-view.html](http://www.colorado.edu/physics/phet/dev/html/ohms-law/1.4.0-dev.12/ohms-law-a11y-view.html)

**Comment by zepumph** [comment]
Alright I added a bit of space for some of the tags I think that is most everything for this issue. @terracoda could you confirm, and make more items to accomplish if you think there is more here. I updated all a11y views on master.

**Comment by zepumph** [comment]
Although obvious in retrospect, adding a line-height to the containing div did not fix everything, I'm still investigating:
![image](https://user-images.githubusercontent.com/6856943/31033967-8aacbe68-a50e-11e7-8c1a-5001c3efb2c4.png)

**Comment by zepumph** [comment]
@terracoda I am pretty bad at css, maybe we could pair sometime on this so that we could do real time changes and make it right. I don't think we will need much time.

**Comment by terracoda** [comment]
Adding `line-height: 1.5;`to the html element should add in enough space. It looks nice when I add that rule dynamically using the inspector tool (see below).
![screen shot 2017-10-03 at 21 17 48](https://user-images.githubusercontent.com/1902116/31154270-8c2f618a-a880-11e7-8989-8d7562f171d9.png)

**Comment by terracoda** [comment]
If you think 1.5 make sit too airy, you can probably go down to 1.4. You do not need a unit on the line-height property.

**Comment by terracoda** [comment]
`line-height` is one of those things you want to define for the entire document, and then only adjust very special cases, with the hope of having no special cases.

**Comment by zepumph** [comment]
One more improvement: 

- [ ] I would like to not have to add the `&accessibility` flag anymore, since we are in the a11y view, it should be assumed that we want a11y enabled and put it in to the iframe src by default.

**Comment by zepumph** [comment]
>I would like to not have to add the &accessibility flag anymore, since we are in the a11y view, it should be assumed that we want a11y enabled and put it in to the iframe src by default.

Just kidding you don't have to do that

**Comment by zepumph** [comment]
@terracoda I implemented the line height that you suggested. I like it a lot. Let me know if there is anything else you would like to see. Please open new issues for future improvements. Closing.

**Comment by terracoda** [comment]
@zepumph looks great!
